grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', [
```

Comment:
\*shrug\*
Since it deals w/ checking strings from the l10n repo, I'd imagine we want to run it closer to when we do a release to make sure the translated markup is valid.